src/features/crops/components/Seeds.tsx

```diff
@@ -140,17 +134,19 @@ export const Seeds: React.FC<Props> = ({ onClose }) => {
         <Button
           disabled={lessFunds() || stock?.lessThan(1)}
           className="text-xs mt-1"
-          onClick={handlBuyOne}
+          onClick={() => buy(1)}
         >
           Buy 1
         </Button>
-        <Button
-          disabled={lessFunds(10) || stock?.lessThan(10)}
-          className="text-xs mt-1"
-          onClick={() => buy(10)}
-        >
-          Buy 10
-        </Button>
+        {bulkSeedBuyAmount > 1 && (
+          <Button
+            disabled={lessFunds(bulkSeedBuyAmount)}
```

Comment:
Wasn't sure if it would be better to also incorporate the players funds, so it would allow you to spend your remaining tokens on as many seeds as you could afford